syft/pkg/dpkg_metadata_test.go

```diff
@@ -86,9 +86,7 @@ func TestDpkgMetadata_FileOwner(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(strings.Join(test.expected, ","), func(t *testing.T) {
-			var i interface{}
-			i = test.metadata
-			actual := i.(FileOwner).OwnedFiles()
+			actual := test.metadata.OwnedFiles()
```

Comment:
Nice! 

What change went in that made so that metadata no longer hast to be cast to `FileOwner` I don't see it in this code diff so did we update something in another repo?

Reply:
Maybe this https://github.com/jonasagx/syft/commit/93d00dc3409ded9a31aa8b52b0512d96bc252f6b

However, if `test.metadata` implements [FileOwner](https://github.com/jonasagx/syft/blob/main/syft/pkg/file_owner.go#L7) there is no need to cast it. [Deep equal](https://github.com/jonasagx/syft/blob/main/syft/pkg/dpkg_metadata_test.go#L90) accepts `interface{}` so any type can be compared.

PS: <3 hast